packages/clay-css/src/scss/components/_stickers.scss

```diff
@@ -47,70 +8,52 @@
 }
 
 .sticker-overlay {
-	align-items: center;
-
-	@include border-radius($sticker-border-radius);
-
-	bottom: 0;
-	display: flex;
-	justify-content: center;
-	left: 0;
-	overflow: hidden;
-	position: absolute;
-	right: 0;
-	top: 0;
+	@include clay-css($sticker-overlay);
```

Comment:
Hey @pat270 it's no big deal but just to understand some of the APIs that we make available and it seems to be a behavior that we do in all components. I see that I have two ways to customize the overlay, using the `$sticker-overlay` variable directly or the `$sticker` variable what's the difference we have here?

I see that the `$sticker-overlay` is not in the scope of the `sticker` class but if I use the variable `$sticker` it is in the scope of the class. Also, I know that with the `$sticker` variable you will be able to modify all the properties besides the overlay here, in that case, it would just be a matter of flexibility to not have to change the whole `$sticker` just to use the overlay?

Reply:
`$sticker-overlay` is for the base `.sticker-overlay` component. I couldn't include it all in `$sticker` because of the specificity issue we ran into with alerts. For example:

```scss
$sticker: (
  sticker-overlay: (
    display: none,
  ),
);
```

will output:

```scss
.sticker .sticker-overlay { display: none; }
```

Whereas the base class is just `.sticker-overlay`. We can nest it and everything will work in Clay CSS. I'm just worried about older implementations again.